css/excercise_one/part-a/excercise_one.css

```diff
@@ -0,0 +1,315 @@
+* {
+  color: black;
+  font-family: 'Poppins',  sans-serif;
+  text-decoration: none;
+}
+
+.card {
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 12px;
+  margin-left: 12px;
+  opacity: 2;
+  position: relative;
+}
+
+.card-text {
+  color: white;
+  font-size: 18px;
+  font-weight: 500;
+  letter-spacing: -1px;
+  text-align: center;
+}
+
+.clients {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  gap: 60px;
+  height: 238px;
+  left: 20%;
+  padding-bottom: 200px;
+  padding-top: 200px;
+  position: absolute;
+  right: 20%;
+}
+
+.clients-logos {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-bottom: 50px;
+  margin-top: 300px;
+}
+
+.clients-phrase {
+  font-size: 30px;
+  font-weight: 500;
+  letter-spacing: -1px;
+  line-height: 32px;
+  padding-bottom: 70px;
+  text-align: center;
+}
+
+.corner-logo {
+  height: 25px;
+  margin-right: 7px;
+  width: 17px;
+}
+
+.corner-title {
+  font-size: 18px;
+  font-weight: 400;
+  letter-spacing: -1px;
+  line-height: 19px;
+  text-align: left;
+}
+
+.desired-users {
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 19px;
+  letter-spacing: -1px;
+  text-align: left;
+}
+
+.events-photo {
+  height: 210px;
+  width: 240px;
+}
+
+.first-line {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  gap: 34px;
+  width: 100%;
+}
+
+.first-line img {
+  flex: 1;
+  object-fit: contain;
+  width: 10%;
+}
```

Comment:
You can write this in a different way. I recommend you to read this: https://developer.chrome.com/articles/css-nesting/#nesting-examples